package-safelist.json

```diff
@@ -50,5 +50,7 @@
   "vendor/vendor-prefixed/autoload-classmap.php",
   "vendor/vendor-prefixed/autoload.php",
   "vendor/vendor-prefixed/stellarwp/**/*",
+  "node_modules/php-date-formatter/js/php-date-formatter.js",
+  "node_modules/php-date-formatter/js/php-date-formatter.min.js",
```

Comment:
Can we include those outside of node_modules ?

Reply:
I think the build system should support this, changing the way dependencies are  managed, in `node_modules` and through `npm` to go with something that is fixed (in `vendor` or `includes`) seems working around a bug.

Furthermore, this is both included directly, and imported in the blocks.

Could the build system be fixed?